docker-compose.yml

```diff
@@ -8,19 +8,29 @@ services:
       - "8080:8080"
     environment:
       fhir_version: 'R4'
-      hapi.fhir.server_address: 'http://localhost:8080/baseR4'
       spring.datasource.url: 'jdbc:postgresql://hapi-fhir-postgres:5432/hapi'
       spring.datasource.username: admin
       spring.datasource.password: admin
       spring.datasource.driverClassName: org.postgresql.Driver
       spring.jpa.properties.hibernate.dialect: ca.uhn.fhir.jpa.model.dialect.HapiFhirPostgres94Dialect
+      spring.config.location: classpath:/application-custom.yaml
```

Comment:
Since we use this custom yaml file for all of our deployed servers, it's important that we use it here for local testing.